### PR TITLE
Add metadata

### PR DESCRIPTION
**Pull request by ellsclytn**
OpenGraph, canonical things, etc.

### CODE REVIEW

File: source/routes/Home/index.js [hunk 1]
```diff
@@ -86,22 +86,39 @@ const Root = ({
...
-    about = { sections: [] },
+    about,
     inspiration,
     getstarted,
     register
-  } = home || {}
+  } = home
+
+  const {
+    opengraph,
```
Comment (kizza) [inline source/routes/Home/index.js]: `opengraph` vs `openGraph` (oh... I see `getstarted`... but I also see `tourTracker`)

File: source/routes/Home/index.js [hunk 2]
```diff
@@ -86,22 +86,39 @@ const Root = ({
...
+    meta
+  } = config
 
   return (
     <TraitsProvider traits={traits}>
       <div>
-        <Helmet
-          title='Cycling Cares'
-        />
+        <Helmet>
```
Comment (kizza) [inline source/routes/Home/index.js]: Now that this is a larger thing... what would your thoughts be about create a dumb component within this file `Header`? that receives `config` and can nicely destructure everything?

File: source/routes/Home/index.js (hunk 1 above)
Comment (ellsclytn) [reply source/routes/Home/index.js]: A prismic nuance. We can only use lowercase within Prismic, and it doesn't seem worth it to go through and manually camelcase things after they come in.

Comment (kizza) [reply source/routes/Home/index.js]: Totally fair

**Pull request merged**